presentation/src/main/java/com/yapp/bol/presentation/view/home/rank/HomeRankFragment.kt

```diff
@@ -255,7 +260,17 @@ class HomeRankFragment : BaseFragment<FragmentHomeRankBinding>(R.layout.fragment
 
     private fun setFloatingButton() {
         binding.btnCreateGroup.setOnClickListener {
-            // todo create match code needed
+            Intent(requireContext(), MatchActivity::class.java).also {
+                it.putExtra(NewGroupActivity.GROUP_ID, viewModel.groupId)
+                startActivity(it)
+            }
+        }
+    }
+
+    private fun setHelpButton() {
+        binding.btnHelp.setOnClickListener {
+            val url = binding.root.context.resources.getString(R.string.home_help_url)
```

Comment:
```suggestion
            val url = binding.root.resources.getString(R.string.home_help_url)
```
p5 : view 에도 `getResource` 함수가 있어 context 로 호출 안해도 사용 가능해요. 정말 별거 아니지만..

Reply:
@ckrudals 감사합니다 수.완! d956eca